lib/css/atomic/_stacks-typography.less

```diff
@@ -151,6 +151,26 @@ h1, h2, h3, h4, h5, h6, p {
     overflow: hidden;
 }
 
+.v-truncate-fade-sm {
+    .truncation-fade(0)
+}
+
+.v-truncate-fade-md {
+    .truncation-fade(1)
+}
+
+.v-truncate-fade-lg {
+    .truncation-fade(2)
+}
+
+.truncation-fade(@count) {
+    -webkit-mask-image: linear-gradient(180deg,#000 (pow(2, @count) * 100px - 50px),transparent); 
```

Comment:
```suggestion
    -webkit-mask-image: linear-gradient(180deg, #000 (pow(2, @count) * 100px - 50px), transparent); 
```

Reply:
This is clever, but we may want to tweak things a bit to make the bottom edge of each fade more consistent. See how you can see more or less of the last row of text on each of those variations? It'd be nice if they were more consistent. My favorite fade is on the small variation :v:

Reply:
I don't have a great trick to make it fade nicely like the small variation. We would need to make sure that `max-height` is set to some multiple of our line height, but the use case I'm imagining is user content with headings, images, tables, etc.

Maybe setting the 100/200/400px to values that are some multiple of our line height so our base case is okay? I'm not sure what font size we'd want to base this off of, though. 🤔 Maybe we can set up different `max-height`/`content-mask` for different sizes? Eg.

```
.v-truncate-fade-sm {
    .truncation-fade(0, 13px)
}

.v-truncate-fade.fs-body2 {
    .truncation-fade(0, 14.3px)
}

.v-truncate-fade.fs-body3 {
    .truncation-fade(0, 15.4px)
}

}

.truncation-fade(@count, @font-size){
   mask-image: linear-gradient(180deg, #000 (@font-size * 12 * @count), transparent);
}
```

Does this spark any ideas for you @aaronshekey?

Reply:
We could try monkeying with `ems` which is based on the font size. Unfortunately, this is a fork and not a branch, so I can't easily pull it down to commit. You've got write access to the repo, by the way, for future stuff.

```
  .v-truncate-fade {
    mask-image: linear-gradient(180deg, #000 4em, transparent);
}
```
Otherwise, I'm totally cool with some magic numbers here if it comes down to that 😛

Reply:
`em` was a good choice, I combined it with the `@lh-md` class so that I could make `max-heigh` based off of number of lines. We could mess with more modifiers so that there's different versions for different line heights, but that feels like too early of an optimization for me!

```
-webkit-mask-image: linear-gradient(180deg, #000 @lh-md * 1em * 9, transparent); 
mask-image: linear-gradient(180deg,#000 @lh-md * 1em * 9,transparent); 
max-height: @lh-md * 1em * 12;
```

![image](https://user-images.githubusercontent.com/6742479/109533278-e57b8200-7a6e-11eb-8bb3-efa839d0710e.png)